content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/rpc-permission-templates.md

```diff
@@ -0,0 +1,98 @@
+---
+date: '2023-01-10'
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-rpc-permissions-templates
+    weight: 6000
+    parent: corda-5-cli-reference
+section_menu: corda-5-beta
+title: "RPC permission templates"
+---
+
+## Create Common RPC Permission Templates
+
+Currently, when the C5 Cluster starts there is a single "admin" account auto-created which can do everything. While this account can be used to perform any action, there is room for error if not used carefully. The idea is to create fine-grained roles for specific actions such as:
+
+• A dedicated role which can create users, roles, permissions and drive all the associations between them;
+• A dedicated role with a set of all the necessary permissions to create a virtual node (including CPI upload);
+• A dedicated role which allows flows to run on this vNode.
+
+{{< note >}}
+These roles and permissions enable certain common operations. Creation of users and associating such users to the roles should be still done manually by an admin user.
+{{< /note >}}
+
+UserAdminRole- Creates roles, permissions, and controls all associations between the user, roles, and permissions. Create this role at cluster bootstrap by the admin user. There is no direct write into DB like previously done with the admin user.
```

Comment:
I can't quite follow this an the following lines. Seems like it should be a table or subsections maybe?